src/cares_wrap.cc

```diff
@@ -589,9 +590,10 @@ void ChannelWrap::EnsureServers() {
 
 class QueryWrap : public AsyncWrap {
  public:
-  QueryWrap(ChannelWrap* channel, Local<Object> req_wrap_obj)
+  QueryWrap(ChannelWrap* channel, Local<Object> req_wrap_obj, const char* name)
       : AsyncWrap(channel->env(), req_wrap_obj, AsyncWrap::PROVIDER_QUERYWRAP),
-        channel_(channel) {
+        channel_(channel),
+        trace_name_(name) {
```

Comment:
does this change cause any external JS API interface change? I guess not.

Reply:
No, this is all internal.